flink-runtime/src/test/java/org/apache/flink/runtime/security/token/DefaultDelegationTokenManagerTest.java

```diff
@@ -43,37 +46,45 @@ public class DefaultDelegationTokenManagerTest {
     @BeforeEach
     public void beforeEach() {
         ExceptionThrowingDelegationTokenProvider.reset();
+        ExceptionThrowingDelegationTokenReceiver.reset();
     }
 
-    @AfterAll
-    public static void afterAll() {
+    @AfterEach
+    public void afterEach() {
```

Comment:
This is just a minor improvement which is not related. We need to do reset after each test and not after class.